platforms/platform_db/platform_defaults.json

```diff
@@ -59,10 +48,106 @@
                      "is the virtual channel number.)  Maximum bandwidth is typically a",
                      "function of the number lines in flight, indepent of request sizes."
                   ],
-               "max-bw-active-lines-c0": "{ 768, 512, 256, 256 }",
+               "max-bw-active-lines-c0": "{ 512, 512, 256, 256 }",
                "max-bw-active-lines-c1": "{ 512, 256, 128, 128 }",
 
-               "max-outstanding-mmio-rd-reqs": 32
+               "max-outstanding-mmio-rd-reqs": 32,
+
+               "comment":
+                  [
+                     "CCI-P requires that an AFU register both the request and response",
+                     "signals before any combinational logic.  On some platforms, it",
+                     "is easier to meet timing when there is more than one register",
+                     "stage.  num-timing-reg-stages is a platform-specific recommendation",
+                     "to AFUs, suggesting the number of registered CCI-P request and",
+                     "response stages for timing across the partial reconfiguration boundary.",
+                     "",
+                     "Note: There is no extra platform-side buffering added for handling",
+                     "extra almost full signals!  Extra buffering beyond one stage",
+                     "counts against the sending limit following almost full.  Typically,",
+                     "2 * (num-timing-reg-stages - 1) slots are lost to buffering.",
+                     "This accounts for the added latency of receiving almost full and",
+                     "also accounts for extra in-flight requests beyond the CCI-P",
+                     "allotted single stage."
+                  ],
+               "num-timing-reg-stages": 1,
+
+               "comment":
+                  [
+                     "This field requests that the platform register CCI-P requests",
```

Comment:
I don't understand the difference between num-timing-reg-stages and num-auto-timing-reg-stages. Why do you set it to null here?

Reply:
num-timing-reg-stages is a platform characteristic. num-auto-timing-reg-stages is a request to actually insert stages. I'll try to clarify on a later commit. There will be some. :-)

Reply:
Let me get it straight with an example.
If I set num-timing-reg-stages to 2, and num-auto-timing-reg-stages to null, no stages will be inserted
If I set num-timing-reg-stages to 2, and num-auto-timing-reg-stages to auto, 2 stages will be inserted
If I set num-timing-reg-stages to 2, and num-auto-timing-reg-stages to 4, 4 stages will be inserted

Is that what is expected?